test/ui-test/testSuites/suite_status/tst_ChatFlow/test.feature

```diff
@@ -50,6 +50,11 @@ Feature: Status Desktop Chat
          Then the user can delete the message at index 0
          Then the last message is not the random message
 
+    Scenario: The user is able to use emoji suggestions
+         When user joins chat room automation-test
+         When the user types "hello :thumbs"
+		 Then the user selects emoji in the suggestion list
+
```

Comment:
I think it's missing some kind of validation at the end. Maybe you could add `the user pressed Enter` or `the user send the written message`
and `then the message EMOJI is displayed in the last message` (this one already exists, just need to use the right words)

Reply:
I agree on that!